scripts/fmt.sh

```diff
@@ -10,4 +10,5 @@ gofmt -w                                                                      \
     $ROOT/ipinfo                                                              \
     $ROOT/grepip                                                              \
     $ROOT/cidr2range                                                          \
-    $ROOT/range2cidr
+    $ROOT/range2cidr                                                          \
+    $ROOT/range2ip                                                            
```

Comment:
There's a bunch of trailing spaces on the last one, which I think you added by accident.